extensions-core/lookups-cached-global/src/main/java/io/druid/query/lookup/NamespaceLookupExtractorFactory.java

```diff
@@ -95,7 +96,12 @@ public NamespaceLookupExtractorFactory(
   public boolean start()
```

Comment:
Most places in druid treat interruptions as fatal. I was simply carrying that on here. If we assume that the locks DO need to be interruptible, do you have a preferred method of communicating that back up the chain, given that InterruptedException isn't in the method signature?